### ISSUE TITLE
error in dependecies while installin iTorch

### ISSUE THREAD

**Issue by meharshjoshi93** [facebook/iTorch#51]
installed Torch
installed all packages and dependencies : even lists them in "luarocks list"

but getting following error:
Missing dependencies for itorch:
env 
image 
torch >= 7.0
![error](https://cloud.githubusercontent.com/assets/6691677/7897015/89f8e1b4-0681-11e5-95e1-251a1b8174a2.png)
please help

**Comment by xksteven** [comment]
I get this same error but I am also able to run itorch and itorch notebook so far without issue.  So I am not sure why this is happening.

**Comment by meharshjoshi93** [comment]
@xksteven  yeah I eventually did, but by fluke. I ended up doing a clean reinstalling of all the dependencies as well as the torch and ipython. Though I don't understand what seemed to be the problem. :/
